src/sentry/static/sentry/app/views/organizationDashboard/index.jsx

```diff
@@ -13,51 +14,58 @@ import getProjectsByTeams from 'app/utils/getProjectsByTeams';
 import withTeams from 'app/utils/withTeams';
 import withProjects from 'app/utils/withProjects';
 
-class Dashboard extends React.Component {
+class Dashboard extends AsyncComponent {
```

Comment:
We should use `AsyncView` and define `getTitle()` to set a document title

Reply:
Hmm, this is technically a component and not a view though. Also, is there a reason we'd want to overwrite the default title (the org name) from `OrganizationContext`?

Reply:
I would say it's a view since the router routes to it.

I think org name for title is fine since it's the dashboard/index page, but a reason to overwrite it would be to have more specific doc titles. Right now all of our org "dashboard" views have the same document title.

Reply:
For all intents and purposes, it's a view, but if we don't intend to change the document title, then it doesn't matter.